### PR TITLE
prompte comment added to repo

### PR DESCRIPTION
**Pull request by Meles-Z**
I am asking you to tell me how can i do this

### CODE REVIEW

File: code_review_guidline/main.dart [hunk 1]
```diff
@@ -31,11 +31,16 @@ class GuessingGame {
     player.GussNumber = guassNumber;
     if (player.GussNumber < numberToBeGuessed) {
       player.isLower();
+      player.isWin = false;
     } else if (player.GussNumber > numberToBeGuessed) {
       player.isGreater();
-    } else {
+      player.isWin = false;
+    } else if (player.GussNumber == numberToBeGuessed) {
```
Comment (Bikila123) [inline code_review_guidline/main.dart]: your code looks good! consider the following
-use lowerCamelCase for  your GussNumber declaration
and your code has some typo errors

Comment (Meles-Z) [reply code_review_guidline/main.dart]: solved for camelCase

Comment (Bikila123) [reply code_review_guidline/main.dart]: the class where in the correct format, just consider for class properties

Comment (Meles-Z) [reply code_review_guidline/main.dart]: solved for typo error

**Pull request merged**